_rules/form-control-non-empty-accessible-name-e086e5.md

```diff
@@ -162,58 +160,34 @@ The implicit label is not supported on `div` elements.
 
 #### Failed Example 6
 
-The explicit label is not supported on `div` elements.
+This element with a `textbox` [role][semantic role] has an empty (`""`) [accessible name][]. The `label` element does not give it an [accessible name][], this only works for native form controls.
 
 ```html
 <label for="lastname">first name</label>
 <div role="textbox" id="lastname"></div>
 ```
 
-#### Failed Example 7
-
-The [accessible name][] is empty.
-
-```html
-<label> <input /></label>
-```
-
-#### Failed Example 8
-
-This `input` element has an [explicit role][] of `none`. However, it is [focusable][] (by default). Thus it has a [semantic role][] of `textbox` due to [Presentational Roles Conflict Resolution][]. It has an empty [accessible name][].
-
-```html
-<input role="none" />
-```
-
 ### Inapplicable
 
 #### Inapplicable Example 1
 
-Hidden to everyone.
+This `input` element is not [included in the accessibility tree][] because of its `style` attribute.
```

Comment:
It's not actually because of `style` attribute alone but because of `display: none;`
```suggestion
This `input` element is not [included in the accessibility tree][] because of its `style` attribute with `display:none;`.
```

Reply:
Or... "display property value set to none".